cmd.c

```diff
@@ -743,12 +743,11 @@ static int cmd_loadApp(phfs_conf_t *phfs, const char *name, const char *imap, co
 
 void cmd_app(char *s)
 {
-	u16 cmdArgsc = 0;
-	int i = 0, argID = 0;
+	u16 i, cmdArgsc = 0, argID = 0;
```

Comment:
`int` type offers the best performace on each CPU (excluding 8-bits systems, where int is 16 bits long), so I see no point in changing `i` and `argID `types from `int `to `u16`.

Reply:
Reverted to previous version.